src/usage/usage.service.spec.ts

```
@@ -13,3 +27,420 @@ describe('calculateResolutionRate', () => {
+  where(_condition: unknown): this {
+    return this;
+  }
+
+  groupBy(_columns: unknown): this {
+    return this;
+  }
+
+  then<TResult1 = unknown[], TResult2 = never>(
+    onfulfilled?:
+      | ((value: unknown[]) => TResult1 | PromiseLike<TResult1>)
+      | null,
```

ESLint

[error] Replace `|·((value:·unknown[])·=>·TResult1·|·PromiseLike<TResult1>)⏎·····` with `((value:·unknown[])·=>·TResult1·|·PromiseLike<TResult1>)`

(prettier/prettier)